src/components/NoteWhiteArea.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { note } from './NoteWhiteArea.scss';
+
+const NoteWhiteArea = ({ className, children, ...props }) =>
```

Comment:
I changed name according to `HelpContainer`. 

But I think that we should allow setting `className` or `props`. 
For `props` it will allow to directly setting events like onClick etc. 
`className` - I'm styling only outline div (rounded corners, padding and shadows). Styles of content is inherited from parents. I can imagine case when you want to style inside of hint a bit differently or set some extra css attributes. Its a bit similar to our problem with material-ui components.